src/Rematch2.jl

```diff
@@ -1,8 +1,8 @@
 module Rematch2
 
-export @match, @match2, MatchFailure
+export @match, @match2, MatchFailure, @match_return, @match_fail
 
-using MacroTools: MacroTools, @capture
+using MacroTools: MacroTools
```

Comment:
out of curiosity, why stop using `@capture`? was there a correctness/performance problem?

Reply:
@capture makes it difficult to track line numbers during traversal.  This is faster and hardly much more complex, so I decided to drop the dependency in its entirety.